source/code-snippets/versioned-api/options.js

```diff
@@ -0,0 +1,7 @@
+// begin serverApiVersion
+const { MongoClient, ServerApiVersion } = require("mongodb");
+
+const uri = "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
+const client = new MongoClient(uri, { serverApi: { strict: true, deprecationErrors: true, version: ServerApiVersion.v1 }});
```

Comment:
Nit: I think its easier to read if each option was on it's own line,

Reply:
Good suggestion, thanks